flink-runtime/src/main/java/org/apache/flink/runtime/asyncprocessing/AsyncExecutionController.java

```diff
@@ -377,7 +392,7 @@ public void triggerIfNeeded(boolean force) {
         stateRequestsBuffer.advanceSeq();
     }
 
-    private void seizeCapacity() {
+    private void seizeCapacity(boolean allowOverdraft) {
```

Comment:
I am not sure in what sense this is seizing the capacity. I do like all the comments - maybe an additional sentence at the start of the comment or in Javadoc to indicate the intent of the the method at a high level would help.

Reply:
This is a private method with detailed comments, so I thought javadoc is not necessary. But I will add one.